stellar/docs/link-token.md

```diff
@@ -109,21 +109,26 @@ ts-node evm/interchainTokenFactory \
 
 **Chain B (Stellar):**
 
-**Note:** On Stellar, we're deploying a customized pre-deployed token contract that you control directly, in contrast to the ITS-managed token on EVM.
+**For Production Use:**
+Use your existing Stellar token contract address. Most users will have their own token already deployed.
 
-**Option 1 - Deploy a custom InterchainToken contract:**
+**For Testing/Demo Purposes Only:**
```

Comment:
Can we make testing specific instructions into a collapsible section so the main user (external devs) don't get distracted by it (for the entirety of the doc)?